src/components/ChattingScreen.js

```diff
@@ -1,31 +1,135 @@
-import React from 'react';
-import styled from 'styled-components';
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
 
-import Header from './Header';
-import MessageSender from './MessageSender';
+import Header from "./Header";
+import MessageSender from "./MessageSender";
 
 export default function ChattingScreen() {
-	const EUNKO = 'https://img.techpowerup.org/200908/eun.png';
-	const COOL = 'https://img.techpowerup.org/200908/NjRiY2JjOGU5YzQz.png';
-
-	const MSGLIST = [
-		{ user: true, content: '안녕하세요 12기 프론트엔드 개발자분들' },
-		{ user: true, content: '저희의 대화를 마음껏 조작해보세요 💌' },
-		{ user: true, content: '상단에 프로필을 눌러서 발신자 변경하면 됩니당~' },
-		{ user: false, content: '안녕 은아' },
-		{ user: false, content: '뭐해 ?' },
-		{ user: true, content: '시원아 넌 최고의 팀장이야' },
-		{ user: false, content: '나도 아니까  ' },
-		{ user: false, content: '그만 말해줘도 돼' },
-		{ user: true, content: 'ㅠㅠ' },
-	];
-
-	return (
-		<Wrapper>
-			<Header></Header>
-			<MessageSender></MessageSender>
-		</Wrapper>
-	);
+  const EUNKO = "https://img.techpowerup.org/200908/eun.png";
+  const COOL = "https://img.techpowerup.org/200908/NjRiY2JjOGU5YzQz.png";
+
+  const MSGLIST = [
+    { user: true, content: "안녕하세요 12기 프론트엔드 개발자분들" },
+    { user: true, content: "저희의 대화를 마음껏 조작해보세요 💌" },
+    { user: true, content: "상단에 프로필을 눌러서 발신자 변경하면 됩니당~" },
+    { user: false, content: "안녕 은아" },
+    { user: false, content: "뭐해 ?" },
+    { user: true, content: "시원아 넌 최고의 팀장이야" },
+    { user: false, content: "나도 아니까  " },
+    { user: false, content: "그만 말해줘도 돼" },
+    { user: true, content: "ㅠㅠ" },
+  ];
+
+  const [messageList, setMessageList] = useState(MSGLIST); //intial state값을 MSGLIST로
+  const [newMessage, setNewMessage] = useState({
+    user: false,
+    content: "",
+  });
+
+  const changeInputMessage = (e) => {
+    const { name, value } = e.target; //name으로 구별, value는 값 자체
+    setNewMessage({
+      ...newMessage,
+      [name]: value,
+    });
+  };
+
+  const addMessage = (e) => {
+    e.preventDefault();
+    if (newMessage.content) {
+      setMessageList([...messageList, newMessage]);
+    } else {
+      alert("메시지를 입력하세요");
+    }
+    setNewMessage({ ...newMessage, content: "" });
+  };
+
+  const changeUser = (e) => {
+    e.preventDefault();
+    if (!newMessage.user) {
+      //현재 시원님인 상태
+      return setNewMessage({
+        user: true, //누르면 고은님으로 change
+        content: newMessage.content,
+      });
+    } else {
+      return setNewMessage({
+        user: false,
+        content: newMessage.content,
+      });
+    }
+  };
+
+  useEffect(() => {
+    window.scrollBy(0, window.innerHeight);
+    window.scrollBy({ behavior: "smooth" });
+  }, [messageList]); //messageList가 변경됐을때 스크롤 내림
+
+  return (
+    <Wrapper>
+      <Header {...{ changeUser }} {...{ newMessage }}></Header>
```

Comment:
추가로, user와 imgUrl props를 전달해주면 header에서 더 간단하게 구현할 수 있을 것 같아용
->`Header.js`도 확인해주세용

```suggestion
      <Header 
      {...{ changeUser }} 
      {...{ newMessage }} 
      user={newMessage.user ? '고은' : '정쿨'}
      imgUrl={newMessage.user ? EUNKO : COOL}></Header>
```

Reply:
아하 그렇겠군요 굳이 url 선언 없이 props에서 전달하면 훨씬 간편할 것 같아요